README.md

```diff
@@ -150,85 +150,87 @@ Non-goals:
 
 # Proposed Solution
 
-`AsyncContext` are designed as a value store for context propagation across
+`AsyncContext.Variable` are designed as a value store for context propagation across
```

Comment:
Some editorial nits:
- I think we can still refer to `AsyncContext` here, since it's both `Variable` and `Snapshot` that are used to propagation.
- In the laster sections, I don't think we need to refer the full `AsyncContext.X` namespace name, we can just say `X` (except in actual code, we should still refer to the namespace).

Reply:
Yeah, omitting the namespace can be more succinct. Updated.